lms/djangoapps/mobile_api/users/tests.py

```diff
@@ -61,13 +62,14 @@ def test_success(self):
 
 
 @ddt.ddt
-class TestUserEnrollmentApi(UrlResetMixin, MobileAPITestCase, MobileAuthUserTestMixin):
+class TestUserEnrollmentApi(UrlResetMixin, MobileAPITestCase, MobileAuthUserTestMixin, MobileCourseAccessTestMixin):
     """
     Tests for /api/mobile/v0.5/users/<user_name>/course_enrollments/
     """
     REVERSE_INFO = {'name': 'courseenrollment-detail', 'params': ['username']}
     ALLOW_ACCESS_TO_UNRELEASED_COURSE = True
     ALLOW_ACCESS_TO_MILESTONE_COURSE = True
+    ALLOW_ACCESS_TO_NON_VISIBLE_COURSE = True
```

Comment:
Good question.  I needed to add this because I found (through code coverage reports) that `verify_failure` and `verify_success` were never actually called.  And that's why I added `MobileCourseAccessTestMixin` as a super class, which exercises those methods.